### PR TITLE
Replaced using of database view

### PR DESCRIPTION
**Pull request by sponkrashin**
With direct query to override some conditions.

### CODE REVIEW

File: server/Arcadia.Assistant.CSP/CspEmployeeQuery.cs [hunk 1]
```diff
@@ -10,7 +10,14 @@ public class CspEmployeeQuery
     {
         private readonly ArcadiaCspContext ctx;
 
-        public const string ArcadianEmployeeQuery = "SELECT * FROM dbo.ArcadianEmployee WHERE IsDelete <> 1";
+        public const string ArcadianEmployeeQuery = @"
+            SELECT
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/CspEmployeeQuery.cs]: rewrite using linq.

File: server/Arcadia.Assistant.Web/Infrastructure/JwtEventsHandler.cs [hunk 2]
```diff
@@ -10,6 +10,12 @@ public class JwtEventsHandler : JwtBearerEvents
     {
         private readonly ILogger logger = LogManager.GetLogger("Auth");
 
+        public override Task TokenValidated(TokenValidatedContext context)
+        {
+            this.logger.Warn($"JWT token validated. User identity: {context.Principal.Identity.Name}");
```
Comment (santee) [inline server/Arcadia.Assistant.Web/Infrastructure/JwtEventsHandler.cs]: why WARN??

Comment (sponkrashin) [reply server/Arcadia.Assistant.Web/Infrastructure/JwtEventsHandler.cs]: Debug artifact... Fixed.

File: server/Arcadia.Assistant.CSP/CspEmployeeQuery.cs (hunk 1 above)
Comment (sponkrashin) [reply server/Arcadia.Assistant.CSP/CspEmployeeQuery.cs]: Done.

**Pull request merged**